forward_command_controller/src/multi_interface_forward_controller.cpp

```diff
@@ -30,15 +30,16 @@ MultiInterfaceForwardController::MultiInterfaceForwardController()
 controller_interface::return_type
 MultiInterfaceForwardController::init(const std::string & controller_name)
 {
-  auto ret = ControllerInterface::init(controller_name);
+  auto ret = ForwardCommandController::init(controller_name);
   if (ret != controller_interface::return_type::SUCCESS) {
     return ret;
   }
 
   try {
-    get_node()->undeclare_parameter("joints");
-    get_node()->undeclare_parameter("interface_name");
-    get_node()->declare_parameter<std::vector<std::string>>("joints_interfaces", {});
+    node_->undeclare_parameter("joints");
+    node_->undeclare_parameter("interface_name");
+    node_->declare_parameter<std::vector<std::string>>(
+      "joints_interfaces", joints_interfaces_);
```

Comment:
A comment about what's happening here would be great. I don't understand without really digging into it